daprdocs/content/en/operations/resiliency/policies.md

```diff
@@ -91,3 +93,41 @@ spec:
         timeout: 45s
         trip: consecutiveFailures > 8
 ```
+
+##### Override Default Retries
+
+Dapr provides default retries for certain request failures and transient errors.  Within a resiliency spec, you have the option to override Dapr's default retry logic by defining policies with reserved, named keywords (for example, defining a policy with the name `DaprBuiltInServiceRetries`, will override default retries for failures between sidecars via service-to-service requests). Policy overrides are not applied to specific targets. 
```

Comment:
```suggestion
Dapr provides default retries for certain request failures and transient errors.  Within a resiliency spec, you have the option to override Dapr's default retry logic by defining policies with reserved, named keywords. For example, defining a policy with the name `DaprBuiltInServiceRetries`, overrides the default retries for failures between sidecars via service-to-service requests. Policy overrides are not applied to specific targets. 
```

Reply:
Re "Policy overrides are not applied to specific targets.". Is this saying "The default policy overrides are not applied to specific targets, since they have a target already specified."?

Reply:
@msfussell no, this implies that when you override a default policy (using a reserved, name keyword) Dapr knows where to apply that policy. You do not need to add that policy to any targets in the target section below